build.sbt

```diff
@@ -13,11 +14,20 @@ lazy val microservice = Project("ukim-auth-checker-api", file("."))
     scalacOptions += "-Wconf:src=routes/.*:s",
     PlayKeys.playDefaultPort := 10160,
     resolvers += Resolver.jcenterRepo, // Keep the existing resolver
-    resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
+    resolvers += MavenRepository(
+      "HMRC-open-artefacts-maven2",
+      "https://open.artefacts.tax.service.gov.uk/maven2"
+    )
   )
   .settings(CodeCoverageSettings.settings: _*)
   .settings(
     Compile / unmanagedResourceDirectories += baseDirectory.value / "resources",
+    ScoverageKeys.coverageExcludedFiles := "<empty>;Reverse.*;.*filters.*;.*handlers.*;.*components.*;.*config.*;" +
+      ".*BuildInfo.*;.*javascript.*;.*Routes.*;.*GuiceInjector;" +
+      ".*testonly.*;.*models.*;.*job.*;",
+    ScoverageKeys.coverageMinimumStmtTotal := 90,
+    ScoverageKeys.coverageFailOnMinimum := true,
+    ScoverageKeys.coverageHighlighting := true
```

Comment:
Most of this already exists in CodeCoverageSettings.settings. The only thing I think in here that makes sense to add is .*config.* - just add that to CodeCoverageSettings.excludedPackages and then let me know what the result it.